CHANGES.rst

```diff
@@ -20,11 +20,15 @@ Unreleased
     (`#3059`_)
 -   :func:`send_file` supports :class:`~io.BytesIO` partial content.
     (`#2957`_)
+-   Support ``static_url_path`` that ends with a forward slash for Flask
```

Comment:
I think the "for Flask app" isn't needed

Reply:
yeah i also wasn't sure abt that.. let me remove it